Cargo.toml

```diff
@@ -24,14 +23,15 @@ domain = { default-features = false, version = "0.5" }
 domain-resolv = { default-features = false, version = "0.5" }
 futures = { default-features = false, features = ["compat", "io-compat"], version = "0.3.5" }
 ipfs-unixfs = { path = "unixfs" }
-libp2p = { default-features = false, features = ["floodsub", "identify", "kad", "tcp-async-std", "mdns", "mplex", "noise", "ping", "yamux"], version = "0.23" }
+libp2p = { default-features = false, features = ["floodsub", "identify", "kad", "tcp-tokio", "mdns", "mplex", "noise", "ping", "yamux"], version = "0.23" }
```

Comment:
TODO: soon tokio-mdns as well, mdns might still use async-std by default.

Reply:
Good catch! I was looking at the main features in search for other `tokio` ones and found none - `mdns` **does** use `async-std` right now, probably gonna change soon, like you're saying.